Pod/Classes/Mocks/ForgeriesTraitCollections.h

```diff
@@ -0,0 +1,15 @@
+@import UIKit;
+
+#define ExtendTraitEnvironment(specific_class) \
```

Comment:
Would've done this with Swift protocol extensions but couldn't get it figured out.